### DISCUSSION TITLE
Add a button when the delay ends instead of auto-redirecting

### DISCUSSION THREAD

**Discussion by wi0lono** [proginosko/LeechBlockNG#507] [Ideas]
Instead of automatically redirecting to the target page, an (optional) button would create another layer of friction. It would also be nice if there was a counter on the delay page showing how many times you've accessed the site on a particular day.

I was curious as to whether something like this is already being worked on. If not, I don't mind helping add it myself.

**Comment by proginosko** [comment]
Deselect the option "Automatically load blocked page when delay countdown reaches zero" and then you'll have to click on the link to load the target page. Same effect!

**Reply by wi0lono** [reply]
Ah, I didn't see the _Show full options_ button, thanks!
